js/packages/mobile-wallet-adapter-protocol/android/build.gradle

```diff
@@ -132,7 +132,7 @@ def kotlin_version = getExtOrDefault('kotlinVersion')
 dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"  // From node_modules
-  implementation "com.solanamobile:mobile-wallet-adapter-clientlib:0.2.0"
+  implementation "com.solanamobile:mobile-wallet-adapter-clientlib:0.2.1"
```

Comment:
This hasn't been published yet; to verify that this code compiled, I published to my local Maven repo

Reply:
Oooh can you lazytwitter me and share how to do that?